tox.ini

```diff
@@ -0,0 +1,93 @@
+[tox]
+envlist =
+    test-{py36,py37,py38,py39}
+    integration-test-{py36,py37,py38,py39}
+    isort
+    flake8
+    black
+    blackdoc
+    mypy
+
+[testenv:test-py36]
+basepython = python3.6
+extras =
+    tests
+    optional
+deps =
+    scikit-optimize
+    cma
+commands = pytest tests --ignore tests/integration_tests
+
+[testenv:test-py37]
+basepython = python3.7
+extras =
+    tests
+    optional
+deps =
+    scikit-optimize
+    cma
+commands = pytest tests --ignore tests/integration_tests
+
+[testenv:test-py38]
+basepython = python3.8
+extras =
+    tests
+    optional
+deps =
+    scikit-optimize
+    cma
+commands = pytest tests --ignore tests/integration_tests
+
+[testenv:test-py39]
+basepython = python3.9
+extras =
+    tests
+    optional
+deps =
+    scikit-optimize
+    cma
+commands = pytest tests --ignore tests/integration_tests
+
+[testenv:integration-test-py36]
+basepython = python3.6
+extras =
+    tests
+    integration
+commands = pytest -s tests/integration_tests
+
+[testenv:integration-test-py37]
+basepython = python3.7
+extras =
+    tests
+    integration
+commands = pytest -s tests/integration_tests
+
+[testenv:integration-test-py38]
+basepython = python3.8
+extras =
+    tests
+    integration
+commands = pytest tests/integration_tests --ignore tests/integration_tests/test_fastai.py --ignore tests/integration_tests/allennlp_tests/test_allennlp.py
+
+[testenv:integration-test-py39]
+basepython = python3.9
+extras =
+    tests
+    integration
+commands = pytest -s tests/integration_tests
+
+[testenv:flake8]
+deps = flake8
+commands = flake8 .
+
+[testenv:black]
+deps = black
+commands = black --check .
```

Comment:
How about adding the `--diff` option as [here](https://github.com/optuna/optuna/blob/master/.github/workflows/checks.yml#L26)?